src/librbd/Journal.h

```diff
@@ -101,10 +101,11 @@ class Journal : public RefCountedObject {
 
   static void is_tag_owner(ImageCtxT *image_ctx, bool *is_tag_owner,
                            Context *on_finish);
-  static void is_tag_owner(librados::IoCtx& io_ctx, std::string& image_id,
+  static void is_tag_owner(librados::IoCtx& io_ctx, const std::string& image_id,
```

Comment:
Should these changes  be part of this PR seeing that they aren't related to the group mirroring feature?